hadoop-ozone/dist/src/main/smoketest/topology/readdata.robot

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Smoketest ozone cluster startup
+Library             OperatingSystem
+Library             BuiltIn
+Resource            ../commonlib.robot
+
+*** Variables ***
+
+
+*** Test Cases ***
+Read data from previously created key
+    ${output} =         Execute          ozone sh key get /topvol1/bucket1/key1
```

Comment:
```suggestion
    ${output} =         Execute          ozone sh key get /topvol1/bucket1/key1 /tmp/key1
```

The `key get` command requires local filename as second argument, so currently it produces this output (trimmed) and does not really verify ability to read data:

```
Missing required parameter: <fileName>
```

Note that on the second run (for the other rack) it will refuse to overwrite `/tmp/key1`, so

1. delete local file after `key get` OR
2. run test case in different container (eg. `om`) OR
3. use random local filename ([example](https://github.com/apache/hadoop-ozone/blob/912dcf0dcbcfb48952443e54f56b33695c413eee/hadoop-ozone/dist/src/main/smoketest/ozonefs/hadoopo3fs.robot#L29) for random string generation)

Reply:
Well spotted. I will add a random filename using the example you gave.